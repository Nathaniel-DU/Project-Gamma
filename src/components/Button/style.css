/* .mint-button {
    width: 90%;
    background-color: #ADEFD1FF;
    border: black 1px solid;
    
    border-radius: 15px;
    display: block;
    margin: auto;
    color: black;
}  */

.mint-button {
    min-width: 90%;
    cursor: pointer;
    margin-right: 0.25em;
    margin-top: 0.5em;
    /* padding-right: 	0.938em;
    padding-left: 	0.938em; */
    border: none;
    border-radius: 10px;
    background-color: #ADEFD1FF;
    color: black;
    margin: auto;
    display: block;
    margin-bottom: 20px!important;
    padding: 10px;
    border: black 1px solid;
    
  }
   
  button:hover {
    background-color: #4A4E69;
    color: #fefefe;
  } 