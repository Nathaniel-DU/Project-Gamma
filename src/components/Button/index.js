import React from 'react';
import './style.css';
export default function Button({ text }) {
    return <button className='mint-button'>{text}</button>
};