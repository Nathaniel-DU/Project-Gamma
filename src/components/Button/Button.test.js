import React from 'react';
import { shallow } from 'enzyme';
import Button from './index';

describe(`Button`, () => {
    it(`should render correctly in "debug" mode`, () => {
        const component = shallow(<Button debug />);
        expect(component).toMatchSnapshot();
    });
});