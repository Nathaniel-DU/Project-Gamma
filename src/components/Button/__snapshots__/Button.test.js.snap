// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly in "debug" mode 1`] = `
<button
  className="mint-button"
/>
`;
